lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
> Minor: Since this is arguably presentational logic, did you consider making this a view helper method? I think I'm happy with the approach you've taken, because you've talked about ideally wanting to store the source data (as opposed to the partial template name) in the YAML.

@floehopper: It didn't cross my mind to add it as a helper method. The `MarriageAbroadCalculator` felt like the natural place for it given how closely this relates to the other service information the calculator is responsible for. If we are able to store acceptable payment types, instead of names of partials, then I can imagine using the `MarriageAbroadCalculator` to retrieve the data and a new helper method to turn it into a sentence.